master/security/service-accounts.md

```diff
@@ -87,6 +88,32 @@ spec:
   selector: 'app == "db"'
 ```
 
+#### Use Kubernetes RBAC to control service account label assignment
+
+Network policies can be applied to endpoints using selectors that match labels on the endpoint, the endpoint's namespace, or the endpoint's service account. By applying selectors based on the endpoint's service account, you can use Kubernetes RBAC to control which users can assign labels to namespaces. This allows you to separate groups who can deploy pods from those who can assign labels to namespaces.
```

Comment:
"assign labels to namespaces" -> "assign labels to service accounts"

Reply:
Fixed